force_wfmanager/central_pane/plot.py

```diff
@@ -177,6 +191,46 @@ def _update_plot_data(self):
         self._plot_data.set_data('x', self._data_arrays[x_index])
         self._plot_data.set_data('y', self._data_arrays[y_index])
 
+        self.resize_plot()
+
+    def resize_plot(self):
```

Comment:
This could porbably be done using chaco settings, but the automatic versions didn't seem to work (passing tight_bounds=False). There was also an option to provide your own function which takes (x-low, x-high, margin, tight_bounds) and returns the plot range (min, max).
But then it would basically be this function, except you couldn't call it in the case where you wanted to manually reset the plot view.